api/envoy/extensions/transport_sockets/tls/v3/common.proto

```diff
@@ -252,6 +252,9 @@ message TlsCertificate {
 
   // [#not-implemented-hide:]
   repeated config.core.v3.DataSource signed_certificate_timestamp = 5;
+  
+  // The name of this TLS certificate.
```

Comment:
Please clarify how the stat looks like whether this field is set or not, and add a link to the relevant statistics page.